config.py

```diff
@@ -33,6 +33,10 @@ class Config(object):
     DM_SUBMISSIONS_BUCKET = None
     DM_ASSETS_URL = None
 
+    DM_MAILCHIMP_USERNAME = None
```

Comment:
We'll need to get these three new environment variables set for Preview/Staging/Production in credentials/PaaS setup before we merge/release this.

Reply:
I don't 100% know how to do that... do you want to figure it out or shall I?